components/tools/OmeroPy/src/omero/plugins/chgrp.py

```diff
@@ -59,6 +59,17 @@ def _pre_objects(self, parser):
             "grp", nargs="?", type=ExperimenterGroupArg,
             help="""Group to move objects to""")
 
+    def is_admin(self, client):
+        # check if the user currently logged is an admin
+        svc = client.sf.getAdminService()
```

Comment:
For the client connection (`conn = self.ctx.conn(args)`) the current context can be gotten with `get_event_context`. See https://github.com/openmicroscopy/openmicroscopy/blob/v5.3.1/components/tools/OmeroPy/src/omero/cli.py#L1392